interactions/client.py

```diff
@@ -361,6 +361,17 @@ def decorator(coro: Coroutine) -> Callable[..., Any]:
             if name is MISSING:
                 raise InteractionException(11, message="Your command must have a name.")
 
+            __name_is_upper: bool = False
+            for _ in name:
+                if _.isupper():
+                    __name_is_upper = True
+            
+            if __name_is_upper:
+                raise InteractionException(
+                    11,
+                    message="Your command name must not contain uppercase characters (Discord limitation)"
+                )
+
```

Comment:
```suggestion
            __name_is_upper: bool = False
            for _ in name:
                if _.isupper():
                  raise InteractionException(
                      11,
                      message="Your command name must not contain uppercase characters (Discord limitation)"
                  )

```

Reply:
For some reason, I thought this would raise an exception for every uppercase char, but no, that's not how exceptions work. Thanks 👍🏾